tests/cql_query_test.cc

```diff
@@ -3937,15 +3937,13 @@ SEASTAR_TEST_CASE(test_like_operator_bind_marker) {
     });
 }
 
-#if 0 // TODO: Enable when like_matcher handles blank pattern.
 SEASTAR_TEST_CASE(test_like_operator_blank_pattern) {
     return do_with_cql_env_thread([] (cql_test_env& e) {
         cquery_nofail(e, "create table t (s text primary key, )");
         cquery_nofail(e, "insert into t (s) values ('abc')");
         require_rows(e, "select s from t where s like '' allow filtering", {});
     });
```

Comment:
How about an extra row with an empty key to test that it is matched by a blank pattern?